regression/cbmc/havoc_slice/test_whole_array_with_offset.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+test_whole_array_with_offset.c
+
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
+^.*expecting FAILURE: SUCCESS$
+^.*expecting SUCCESS: FAILURE$
+^.*dereference .*: FAILURE$
```

Comment:
I see that you have a different name for each test case (i.e., a `.c` file and a test description). The usual pattern across the regression tests is to create a folder with a descriptive name (like the ones you're using for each test case) and add a `main.c` file and a `test.desc` file inside that folder. At the end of each test description, we add a small explanation of what behavior the test is covering. Not a blocker, but I suggest you to update this test to follow the same pattern.